.github/workflow/code-quality-push.yaml

```diff
@@ -0,0 +1,23 @@
+name: Code Quality
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  codeQuality:
+    runs-on: ubuntu-20.04
+    name: PHP
+    steps:
+      - name: Checkout changes
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup PHP and linter
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.0'
+          tools: phpcs
+      - name: Run code quality checks
+        run: ./.github/workflows/utilities/lint-push ${{ github.sha }}
```

Comment:
You should be able to just run `php vendor/bin/grumphp run --no-interaction` (or to run a specific task `php vendor/bin/grumphp run --no-interaction --tasks=phplint`) instead of providing a utility script.

Reply:
The utility script is for running linting only on the changed files to speed up things. Also the idea is to skip the composer install, which makes it even quicker.
I can see that it's possible to pass in the changed file to grumphp: https://github.com/phpro/grumphp/blob/master/doc/commands.md#run

I will try to change the utility scripts to make them use grumphp.